### PR TITLE
[CMS] feat: add proposal spending summary and details

### PR DESCRIPTION
**Pull request by tiagoalvesdulce**
This PR closes https://github.com/decred/politeiagui/issues/2019

### Solution description

I added two new routes: `/admin/proposalsbilling`  and `/admin/proposalsbilling/:token` that consume data from `spendingsummary` and `spendingdetails` endpoints. Created a new normalized redux state branch called `proposalBilling` to be the single source of truth for this data. This PR is a WIP and will be imrpoved before merged.

### Dependencies

Depends on https://github.com/decred/politeia/pull/1224/

### Summary
Screen Shot 2020-07-01 at 18 09 58

### Details
Screen Shot 2020-07-01 at 18 10 18

### CODE REVIEW

File: src/containers/Invoice/ProposalBillingDetails/ProposalBillingDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+];
+
+const getInvoiceTotal = (rate, lineItems) => {
+  const laborInMinutes = lineItems.reduce((acc, cur) => acc + cur.labor, 0);
+  const laborInHours = laborInMinutes / 60;
+  return laborInHours * rate; // total
+};
+
+const getDetailsData = (invoices) => {
+  const formattedData =
```
Comment (amassarwi) [inline src/containers/Invoice/ProposalBillingDetails/ProposalBillingDetails.jsx:27]: This function can be a one liner :)

File: src/reducers/models/proposalBilling.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import * as act from "src/actions/types";
+import { set } from "lodash/fp";
+
+const DEFAULT_STATE = {
+  proposalsList: null,
+  proposalDetails: null
+};
+
+// const normalizeProposalBilling = (proposals) =>
```
Comment (amassarwi) [inline src/reducers/models/proposalBilling.js:9]: comments

File: src/reducers/models/proposalBilling.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+//   proposals.reduce((acc, cur) => {
+//     return {
+//       ...acc,
+//       [cur.token]: cur
+//     };
+//   }, {});
+
+const proposalBilling = (state = DEFAULT_STATE, action) => {
+  return action.error
```
Comment (amassarwi) [inline src/reducers/models/proposalBilling.js:18]: Also here no need for the return I guess

**Comment by alexlyp** [comment]
Needs a rebase, but  decred/politeia#1224 has been merged to final testing/review may occur.

**Pull request merged**